recipes/corner/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "corner" %}
+{% set version = "2.0.1" %}
+{% set sha256 = "102e22797ee75d1432b6dc66aa2850f61388996ece66fd6600508742d2a7b88f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - matplotlib
+
+test:
+  imports:
+    - corner
+
+about:
+  home: http://github.com/dfm/corner
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Make some beautiful corner plots of samples.'
+
+  description: |
+    corner uses matplotlib to visualize multidimensional samples using a scatterplot matrix.
+    In these visualizations, each one- and two-dimensional projection of the sample is
+    plotted to reveal covariances. corner was originally conceived to display the results
+    of Markov Chain Monte Carlo simulations and the defaults are chosen with this application
+    in mind but it can be used for displaying many qualitatively different samples.
+  doc_url: http://corner.readthedocs.io/
+  dev_url: https://github.com/dfm/corner
+
+extra:
+  recipe-maintainers:
+    - guillochon
+    - pkgw
+    - dfm
```

Comment:
@dfw are you ok being added as a co-maintainer of the `corner` recipe on conda-forge?